File: python/tr19.py
ingresos=float(input("ingrese sus ingresos"))
if ingresos<=1500000:
    impuesto=0
elif ingresos<=3000000:
    impuesto=ingresos *0.10
else:
    ingresos>=3000000
impuesto=ingresos *0.20

neto=ingresos-impuesto
print("\n--- Resultado ---")
print(f"Salario: ${salario:,.0f}")
print(f"Impuesto: ${impuesto:,.0f}")
print(f"Salario neto: ${neto:,.0f}")

File: java script/tr19.js
let salario = prompt("Ingrese su salario");
let impuesto = 0;
if (salario < 1500000) {
    impuesto = 0;
} else if (salario <= 3000000) {
    impuesto = salario * 0.10;
} else {
    impuesto = salario * 0.20;
}
let neto = salario - impuesto;
alert(
    "Salario: " + salario +
    "\nImpuesto: " + impuesto +
    "\nSalario neto: " + neto
);